docs/content_management/configure_ct_field_settings.md

```diff
@@ -4,14 +4,21 @@ description: Configure Field settings in Content Types.
 
 # Configure Content Type Fields
 
-When you [modify a Content Type](create_edit_content_types.md) and add or modify [Fields](content_model.md#fields-and-field-types), 
+When you work with a Content Type to add or modify [Fields](content_model.md#fields-and-field-types), 
```

Comment:
Why is the link to 'Content Type' removed?
```suggestion
When you work with a [Content Type](content_model.md#content-types) and you want to add or modify [Fields](content_model.md#fields-and-field-types), 
```

Reply:
Original link referenced a procedure, I moved this link to the beginning of the procedure, where parts of it were duplicated. I agree, let's add a link to the concept.